internal/generator/vector/output/elasticsearch/cert-key.go

```diff
@@ -14,5 +14,5 @@ func (kc TLSKeyCert) Template() string {
 	return `{{define "` + kc.Name() + `" -}}
 key_file = {{.KeyPath}}
 crt_file = {{.CertPath}}
-{{- end}}`
+{{end -}}`
```

Comment:
I see several place  {{end}} is used like https://github.com/openshift/cluster-logging-operator/pull/1361/files#diff-8af1a5ddfc62a63b6a37333ca27fbda06b0fe1018dd08b23936f19123e5045e2R40

Reply:
in this case we need an end line after `crt_file` line.
`{{- end}}` consumes this end line, and next template is forced to add a line